src/rust/engine/src/nodes.rs

```diff
@@ -1411,10 +1406,12 @@ impl Node for NodeKey {
           .params
           .keys()
           .filter_map(|key| {
-            if key
-              .type_id()
-              .as_py_type(py)
-              .is_subtype_of(py, &engine_aware_param_ty)
+            // TODO: Switch to PyO3's upcoming mechanism for this:
+            // https://github.com/PyO3/pyo3/pull/1985.
```

Comment:
I continue to be very impressed with the PyO3 community! They fixed the issue I flagged within a few hours, and were also very welcoming to my feedback as a relative outsider.

Reply:
What comes around goes around ;)